nodeup/pkg/model/ntp.go

```diff
@@ -46,22 +49,102 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
+	var ntp string
+	switch b.Cluster.Spec.CloudProvider {
+	case "aws":
+		ntp = "169.254.169.123"
+	case "gce":
+		ntp = "time.google.com"
+	default:
+		ntp = ""
+	}
```

Comment:
The code passes a variable named `ntp` to a parameter named `ip`, while there is a different parameter named `ntp`. This would be less confusing if the variables were named consistently.